.kitchen.yml

```diff
@@ -35,6 +35,9 @@ suites:
     - recipe[nrpe::default]
     - recipe[nagios_test::default]
     - role[monitoring]
+    verifier:
+      inspec_tests:
+        - test/smoke/server_package
```

Comment:
You shouldn't need this line, unless I'm misunderstanding what you're trying to do?
https://github.com/sous-chefs/line-cookbook/tree/master/test/integration
https://github.com/sous-chefs/line-cookbook/blob/master/.kitchen.yml